### PR TITLE
Added `fromArray` method

### PR DESCRIPTION
**Pull request by rustamwin**
make collection without create instance

| Q             | A
| ------------- | ---
| Is bugfix?    | yes/no
| New feature?  | yes
| Breaks BC?    | yes/no
| Tests pass?   | yes/no
| Fixed issues  | comma-separated list of tickets # fixed by the PR, if any

### CODE REVIEW

File: src/Collection.php [hunk 1]
```diff
@@ -74,6 +74,15 @@ public function getData()
         return $this->_data;
     }
 
+    /**
+     * @param array $data
+     * @return Collection
+     */
+    public static function from(array $data = [])
```
Comment (cebe) [inline src/Collection.php]: from can be confused with SQL `FROM` as this class may be used also in combination with AR. I suggest to rename it to `fromArray()`.

**Comment by cebe** [comment]
what is the benefit of `new Collection([...])` over `Collection::fromArray([...])` ?

**Comment by rustamwin** [comment]
to allow using without create instance

File: src/Collection.php (hunk 1 above)
Comment (rustamwin) [reply src/Collection.php]: Good point :smile:

**Comment by cebe** [comment]
please explain in more detail, from the syntax you do not write `new` but you hide it in a static method instead of writing the code yourself. what exactly is the benefit, why do we need this method?

**Comment by samdark** [comment]
I don't think there's a need for such method.

**Pull request closed without merging**